src/server/templates/_components/form-textbox.njk

```diff
@@ -39,7 +39,7 @@
     type="text"
     name="{{ name }}"
     id="input-{{ name }}"
-    class="form-textbox{{ ' has-error' if error }}"
+    class="form-textbox{{ ' form-textbox--short' if variant == 'short' }}{{ ' has-error' if error }}"
```

Comment:
Could also express it like this to avoid extra curly braces:
```jinja
class="form-textbox{{ [' form-textbox--short' if variant == 'short', ' has-error' if error] | join }}"
```